src/api/starknet.js

```diff
@@ -1,28 +1,34 @@
-import {ActionTypes} from '../constants/action-types';
-import {generateRandomHex} from '../utils/generate-random-hex';
-import {apiRequest} from './api-request';
+import {promiseHandler} from '@starkware-industries/commons-js-utils';
+import {getStarknet} from 'get-starknet-core';
 
-const path = process.env.REACT_APP_STARKNET_ALPHA_SERVICE_URL;
+const getAccount = async () => {
+  const {account} = await getStarknet().getLastConnectedWallet();
```

Comment:
Why do we call `getLastConnectedWallet` and not just `getStarknet().account` ?

Reply:
This is how getStarknet works in version 2